dotnet/src/SemanticKernel/TemplateEngine/PromptTemplateEngine.cs

```diff
@@ -85,11 +85,22 @@ public async Task<string> RenderAsync(IList<Block> blocks, SKContext context)
             }
         }
 
+        var result = new StringBuilder();
+        foreach (Task<string> t in tasks)
+        {
+            result.Append(await t.ConfigureAwait(false));
+        }
+
         // TODO: remove PII, allow tracing prompts differently
         this._log.LogDebug("Rendered prompt: {0}", result);
         return result.ToString();
     }
 
+    public Task<IList<Block>> RenderCodeAsync(IList<Block> blocks, SKContext executionContext)
+    {
+        throw new System.NotImplementedException();
+    }
+
```

Comment:
What's this for? Now that it's removed from the interface, we shouldn't need this anymore...

Reply:
VSCode autofix added this since I changed the `class` first🤦🏾 removed.